pkgs/applications/editors/emacs/elisp-packages/manual-packages/tree-sitter-langs/update-defaults.py

```diff
@@ -55,7 +55,7 @@ def build_attr(nixpkgs, attr: str) -> str:
 
 if __name__ == "__main__":
     cwd = dirname(abspath(__file__))
-    nixpkgs = abspath(join(cwd, "../../../../../.."))
+    nixpkgs = abspath(join(cwd, "../../../../../../.."))
```

Comment:
This path should point to which file?
To the root default.nix?

Reply:
It should point to root of nixpkgs repo. Unsure on the original reasoning, but without this change, this script fails to execute.